eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -84,6 +84,15 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <configuration>
+          <additionalOptions>-maxLineLength 120</additionalOptions>
```

Comment:
Might be useful to call out why this config is here - for code snippets output.

Reply:
Done